LibreNMS/Alert/RunAlerts.php

```diff
@@ -524,8 +524,12 @@ public function extTransports($obj)
                 $obj['msg']       = $type->getBody($obj);
                 c_echo(" :: $transport_title => ");
                 $instance = new $class($item['transport_id']);
```

Comment:
Maybe include the instance creation too?

Reply:
I can do that if you want me to. I'm not sure if any existing transports have the possible of throwing on creation and API is only one I know of that could throw on delivery.

Reply:
I don't think any really define __construct() right now.

Reply:
I can include just in case? I think the reasoning could apply to both. If a particular transport throws, for whatever reason, continue to try any other transports.

Reply:
And theoretically the DB could throw right? If for some reason the database was unavailable momentarily. Although in that case every transport would fail.